cclib/parser/serenityparser.py

```diff
@@ -46,7 +48,9 @@ def extract(self, inputfile, line):
             self.set_attribute("mult", int(line.split()[1]) + 1)
 
         # Extract from atoms: number of atoms, elements, and coordinates
-        if line.strip().startswith("Current Geometry (Angstrom):"):
+        if line.strip().startswith("Current Geometry (Angstrom):") and not getattr(
+            self, "atomcoords", []
```

Comment:
Why do you need the modified condition? `append_attribute` should already be doing the right thing.